packages/storybook-webpack-angular-types-plugin/src/lib/webpack-angular-types-plugin/type-extraction/ast-utils.ts

```diff
@@ -27,12 +39,25 @@ export function extractComponentOrDirectiveAnnotatedClasses(
 					classDeclaration.getDecorator('Directive') ||
 					classDeclaration.getDecorator('Pipe') ||
 					classDeclaration.getDecorator('Injectable')
-				),
+				) || hasJsDocsTag(classDeclaration.getJsDocs(), 'include-docs'),
```

Comment:
Does this make sense? Are there non Component/Directive/Pipe/Injectable classes that we support?

On the other hand, it does not really hurt to have it here 🤷

Reply:
I thought about adding ArgsTables for DataSources. And also we allow any interface with "@include-docs", so why not classes?